src/components/AddressToScript.tsx

```diff
@@ -12,8 +14,8 @@ declare global {
   }
 }
 if (ExecutionEnvironment.canUseDOM) {
-  require("../../static/lumos.min.js");
-  const lumos = window.lumos;
+  const lumos = require("../../static/lumos.min.js");
+  // const lumos = window.lumos;
```

Comment:
```suggestion
   require("../../static/lumos.min.js");
   const lumos = window.lumos;
```
cause server build different with local build，we need to change back code before push back to server，or we will get `Cannot read properties of undefined (reading 'initializeConfig')` error。